fix(TikTok): Update patches to latest version

Nuckyz opened the pull request:
TODO: Find a way to inject the settings button

PR also does a lot of clean-up

Nuckyz commented:
I need assist on this one. The new code for UI which creates the settings is completely automatically generated by their tools, and each component is composed by very specific code generated exactly for it. It seems extremely hard to use their own components now to add the settings button, so a different approach would probably be better. However, I am sure not how a different approach could be done, as I dont have a lot of experience with UI in specific.

Nuckyz commented:
One approach could be perhaps modifying one of the existing buttons

Nuckyz commented:
I have some new ideas but haven't tried yet. I will come back here once I try.

Nuckyz commented:
Been a little bit busy in the past days, I will continue this as soon as I'm able to.

OldManJenkins12 commented:
Out of curiosity, would patches be bumped to 39.8.1 (version when this PR was created) or the true latest version, which as of the time of typing this is 40.2.2

pikaczu787 commented:
@Nuckyz are you gay?

Nuckyz commented:
> @Nuckyz Any update?

Busy with Spotify for now, this will have to wait.

Nuckyz commented:
> I meant, why not repose that button so it opens the ReVanced settings. Can pick a button that isn't important.
> 
> That stock button would lose its original functionality, but maybe it could be restored by calling into the original code/listener (add a replacement button in the ReVanced dialog that calls the original code).

Alright, I'll keep in that mind for the next time I resume this

Nuckyz commented:
> can confirm this branch works fine on TikTik 39.8.4 and rebased on top of the latest release tag v5.27.0, the lack of settings doesn't really bother me, just changed them in the source thanks!

Some patches still crash so be aware

AykhanUV commented:
@LisoUseInAIKyrios without settings button you can't configure most of the patches, and that makes them unusable too

oSumAtrIX commented:
<img width="343" height="89" alt="image" src="https://github.com/user-attachments/assets/1234ef4a-83b0-4130-ba6a-60ac91963e21" />

OldManJenkins12 commented:
A TikTok version compatibility bump is required, many functions of the app no longer work for me, an update is required
